Kickstarter-iOS/Features/DashboardDeprecationBanner/DashboardDeprecationView.swift

```diff
@@ -0,0 +1,50 @@
+import Library
+import SwiftUI
+
+struct DashboardDeprecationView: View {
+  private let contentPadding = 12.0
+  private let imageSizeMultiplier = 1.5
+  private let deprecationDateText = formatted(dateString: "2023-08-14")
```

Comment:
Remember we're support multiple time zones and languages so we should use conventions to add dates to any presented strings.